fre/catalog/frecatalog.py

```diff
@@ -1,15 +1,20 @@
+'''
+entry point for fre catalog subcommands
+'''
+
 import click
-import catalogbuilder
+#import catalogbuilder
```

Comment:
Isn't this needed?

Reply:
it is not! if it were, the new API tests in `fre/catalog/tests` would be failing

Reply:
I can't disagree with you. But it's weird still isn't it? should we remove the line then?